### PR TITLE
Fix attribute transition

### PR DESCRIPTION
**Pull request by Pessimistress**
#### Background

Attribute transition is throwing `Found instanced attributes on non-instanced model transform-model` error. E.g. `instancePositionsFrom` and `instancePositionsTo` are not supposed to be instanced in the transform feedback model, but they are auto-inferred to be based on their names.

#### Change List
- Stop using attribute descriptors in AttributeTransitionManager.

### CODE REVIEW

File: test/apps/attribute-transition/app.js [hunk 1]
```diff
@@ -83,14 +77,12 @@ class Root extends Component {
...
       <div>
         <DeckGL
-          views={new MapView({id: 'map'})}
           controller={MapController}
           viewState={viewState}
           onViewStateChange={evt => this.setState({viewState: evt.viewState})}
           layers={layers}
         >
           <StaticMap
-            viewId="map"
```
Comment (georgios-uber) [inline test/apps/attribute-transition/app.js]: Is this related?

Comment (Pessimistress) [reply test/apps/attribute-transition/app.js]: No.
